### PR TITLE
A Login Screen.

### PR DESCRIPTION
**Pull request by nqthqn**
TODO:
- [x] Create Login view that authenticates against API 
- [X] Create authorized only Components `KidList` and `KidDetail`
- [ ] Create logout route
- [x] Store auth token in rootScope equivalent / axios for making requests to API with auth headers
- [ ] Skip to login screen with "next view" queued up if `/kids` is accessed while not logged in
- [ ] Save "logged in" state until logout (localStorage? cookies?)

### CODE REVIEW

**Comment by PatrickRBailey** [comment]
this looks great!

**Comment by nqthqn** [comment]
@mtomcal I'm getting these errors if I try to include `<Navigation />` in the render method of `<App />`

https://github.com/lumastories/web-hansel/pull/2/commits/27e0a492db87562e3f4a85efa602938064e20a96#diff-14b1e33d5bf5649597cdc0e4f684daddR25

```
warning.js:36 Warning: React.createElement: type should not be null, undefined, boolean, or number. It should be a string (for DOM elements) or a ReactClass (for composite components). Check the render method of `App`

invariant.js:38 Uncaught Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. Check the render method of `App`.
```

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import React, { Component } from 'react'
-import axios from 'axios'
-
-
 import './App.css';
-import {Navigation, Footer} from './Components.js'
+
+import Navigation from './Components/Navigation'
```
Comment (mtomcal) [inline src/App.js]: `import {Navigation} from './Components/Navigation'`

Comment (nqthqn) [reply src/App.js]: Now I am getting

```
warning.js:36 Warning: React.createElement: type should not be null, undefined, boolean, 
or number. It should be a string (for DOM elements) or a ReactClass 
(for composite components). Check the render method of `NavLink`.
```

from `Navigation.js`:

``` jsx
import React, { Component } from 'react'
import Link from 'react-router'

export class NavLink extends Component {
  render(){
    return( <Link {...this.props} activeClassName="active" />)
  }
}

export class Navigation extends Component {
  render() {
    return (
      <div>
        <nav>
          <NavLink to='/' onlyActiveOnIndex={true}>Home</NavLink>
          <NavLink to='/login'>Login</NavLink>
          <NavLink to='/kids'>Kids</NavLink>
        </nav>
      </div>
    )
  }
}
```

**Comment by nqthqn** [comment]
Going to save the login creds later this is good enough for now

**Pull request merged**